lib/rubocop/cop/sorbet/callback_conditionals_binding.rb

```diff
@@ -52,13 +52,38 @@ def autocorrect(node)
             _, _, block = conditional.child_nodes
             expected_class = node.parent_module_name
 
+            if block.source.include?("T.unsafe(self).")
```

Comment:
Can't it be as problem with this?

```rb
something: -> { T.unsafe(self).passing_splat(*args) }
```

Reply:
If we're going to change working code into non-working code yes.

Another solution is to move the `T.unsafe` part in another cop that is optional and just used to auto-correct the `T.unsafe` in `T.bind`